---
bool success
sensor_msgs/PointCloud2 cloud
sensor_msgs/Image disparity_image
float32 time